soepy/test/test_unit.py

```diff
@@ -362,6 +511,9 @@ def test_educ_level_shares():
     # Set probability of having children to zero for all periods
```

Comment:
Why do we always need this setting them to zero?

Reply:
Not necessarily needed except in unit test that specifically checks that there are no kids when the probabilities of child arrival are zero. Expanded to more general case of nonzero probabilities.